test/interface.jl

```diff
@@ -282,6 +282,94 @@ struct NonInvertibleBijector{AD} <: ADBijector{AD} end
         @test f_t == f_a
     end
 
+    @testset "Stacked <: Bijector" begin
+        # `logabsdetjac` without AD
+        d = Beta()
+        b = bijector(d)
+        x = rand(d)
+        y = b(x)
+        sb = vcat(b, b, inv(b), inv(b))
+        @test logabsdetjac(sb, [x, x, y, y]) ≈ 0.0
+
+        # `logabsdetjac` with AD
+        b = DistributionBijector(d)
+        y = b(x)
+        sb1 = vcat(b, b, inv(b), inv(b))             # <= tuple
+        sb2 = Stacked([b, b, inv(b), inv(b)])        # <= Array
+        @test logabsdetjac(sb1, [x, x, y, y]) ≈ 0.0
+        @test logabsdetjac(sb2, [x, x, y, y]) ≈ 0.0
+
+        @testset "Stacked: ADVI with MvNormal" begin
+            # MvNormal test
+            dists = [
+                Beta(),
+                Beta(),
+                Beta(),
+                InverseGamma(),
+                InverseGamma(),
+                Gamma(),
+                Gamma(),
+                InverseGamma(),
+                Cauchy(),
+                Gamma(),
+                MvNormal(zeros(2), ones(2))
+            ]
+
+            ranges = []
+            idx = 1
+            for i = 1:length(dists)
+                d = dists[i]
+                push!(ranges, idx:idx + length(d) - 1)
+                idx += length(d)
+            end
+
+            num_params = ranges[end][end]
+            d = MvNormal(zeros(num_params), ones(num_params))
+
+            # Stacked{<:Array}
+            bs = bijector.(dists)     # constrained-to-unconstrained bijectors for dists
+            ibs = inv.(bs)            # invert, so we get unconstrained-to-constrained
+            sb = Stacked(ibs, ranges) # => Stacked <: Bijector
+            x = rand(d)
+            sb(x)
+            @test sb isa Stacked
+
+            td = transformed(d, sb)  # => MultivariateTransformed <: Distribution{Multivariate, Continuous}
+            @test td isa Distribution{Multivariate, Continuous}
+
```

Comment:
L339 looks like an emtpy line.

Reply:
Yes, I've just added some whitespace to indicate a separation between the two tests, this and the one that "starts" at L341